src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -86,8 +86,8 @@ internal class TokenAcquisition : ITokenAcquisitionInternal
         private readonly ISet<string> _metaTenantIdentifiers = new HashSet<string>(
             new[]
             {
+                Constants.Common,
```

Comment:
this is the only fix that is needed. Can you add a test as well? or modify an existing test?

Reply:
Sure! Will add it tomorrow.

Reply:
@hajekj Actually, nevermind on the tests, you'll need access to our KeyVault to run the tests that would cover this. If you can take a look at the comment above, and then once we merge this, i will add the test(s). If you can test manually though and share the results (assuming it works), that would be helpful.

Reply:
I added a [branch](https://github.com/AzureAD/microsoft-identity-web/blob/jennyf/addTenantTests/tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs#L74) with some additional tests for the work you have here. will merge after we merge this.

Reply:
Looks good to me!